strawberryfields/device.py

```diff
@@ -35,16 +34,21 @@ class DeviceSpec:
             - modes (int): number of modes supported by the target
             - compiler (list): list of supported compilers
             - gate_parameters (dict): parameters for the circuit gates
+
+        certificate (dict): the certificate of the device
     """
 
-    def __init__(self, spec: Mapping[str, Any]) -> None:
+    def __init__(
+        self, spec: Mapping[str, Any], certificate: Optional[Mapping[str, Any]] = None
```

Comment:
Just curious: if we use `spec`, why not use `cert`?
```suggestion
        self, spec: Mapping[str, Any], cert: Optional[Mapping[str, Any]] = None
```

Reply:
Good point. No reason really. Cert sounds good! 👍